src/main/java/uk/gov/di/ipv/cri/common/library/domain/AuditEvent.java

```diff
@@ -14,6 +20,9 @@ public class AuditEvent {
     @JsonProperty("component_id")
     private final String issuer;
 
+    @JsonProperty("restricted")
+    private Map<String, String> restricted = new HashMap<>();
```

Comment:
what is the purpose / meaning of the `restricted` field?
Would a more appropriate name be `context` or `metadata` ?